src/site/topics/algorithm-analysis/algorithm-analysis.rst

```diff
@@ -386,21 +386,21 @@ Stack Comparisons
     * ``LinkedStack`` vs ``ArrayStack``
 
 
-Popping
--------
+Popping & Peeking
+-----------------
 
 .. literalinclude:: /../main/java/LinkedStack.java
     :language: java
     :linenos:
-    :lineno-start: 32
-    :lines: 32-41
+    :start-after: // [start-pop_peek]
+    :end-before: // [end-pop_peek]
```

Comment:
Are the `//` necessary? Only `[tag]` seems more natural.

Reply:
agreed --- will check tomorrow